crew

```diff
@@ -0,0 +1 @@
+bin/crew
```

Comment:
I don't think this will execute like you might expect.  Better to make it a symlink.

Reply:
it is a symlink

Reply:
github is just displaying it weirdly